Clean Swift/Models.xctemplate/___FILEBASENAME___Models.swift

```diff
@@ -23,7 +35,7 @@ enum ___VARIABLE_sceneName___Models {
         struct ViewModel {
             var exampleVariable: String?
         }
-    }
+    }    
```

Comment:
@@
<img width="798" alt="Screenshot 2019-10-03 at 1 36 02 PM" src="https://user-images.githubusercontent.com/476734/66101881-007c6500-e5e3-11e9-8cd9-700d61d07904.png">
Can we make sure that there will be no trailing whitespace to the template?

Reply:
The template is done via Sublime Text and not with Xcode, so some plugins are needed to auto remove the trailing whitespaces. 

But sure, we can check in Xcode so that  we don’t have trailing whitespaces when we are coding in our codebase :)

Reply:
I added this to sublime:
```
{
	.
	.
	.
	"trim_trailing_white_space_on_save": true
}
```